rolling/distribution.yaml

```diff
@@ -1995,6 +1995,24 @@ repositories:
       url: https://github.com/osrf/osrf_testing_tools_cpp.git
       version: master
     status: maintained
+  ouxt_common:
+    doc:
+      type: git
+      url: https://github.com/OUXT-Polaris/ouxt_common.git
```

Comment:
There's no LICENSE in the repository, though the `package.xml` in the individual packages says "Apache 2.0".  Please put a LICENSE.

But more concerningly, there is really no content in this PR other than two meta-packages.  While that doesn't automatically disqualify it from being added to the index, I have to wonder what the benefit to the community is.  Can you explain more about what the plans here are?